### ISSUE TITLE
ref output misses SliceTiming?

### ISSUE THREAD

**Issue by yarikoptic** [neurolabusc/dcm_qa#4]
"testing" fresh release of dcm2niix v1.0.20170621 and it shows diffs like
```
diff -br -I ConversionSoftwareVersion /home/yoh/deb/gits/pkg-exppsy/dcm_qa/Out/6_8_PF_no_iPAT_2.json /home/yoh/deb/gits/pkg-exppsy/dcm_qa/Ref/6_8_PF_no_iPAT_2.json
27,63d26
<       "SliceTiming": [
<               1.255,
<               0,
<               1.325,
<               0.07,
<               1.395,
<               0.14,
...
```

**Comment by dangom** [comment]
Wow, that's funny. Doesn't happen here. 
I tried both under MacOS and CentOS.

Could it be that your `diff` doesn't accept the `-b` flag?

**Comment by neurolabusc** [comment]
Well spotted. I have released a new version of dcm2niix (v1.0.20170623)  that should consistently report slice timings for all systems and input datasets. I will generate new Ref images shortly. Nice example of how this validator can be useful.
